### PR TITLE
Bonfire whitelist

### PR DESCRIPTION
**Pull request by Litogin**
## About the PR

Adding a whitelist for bonfire

## Why / Balance

In order to place only certain objects into the fire

## Media

https://github.com/crystallpunk-14/crystall-punk-14/assets/139079015/a02bca3d-c19f-4649-a1cd-050995d6f278

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Litogin
- add: Added whitelist for bonfire

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Misc/paper.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@
     - Document
     - Trash
     - Paper
+    - CP14Bonfire # Testing
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Objects/Misc/paper.yml:33]: Just add new CP14 paper proto with parenting from general paper. Dont modify ss14 content please

Comment (TheShuEd) [reply Resources/Prototypes/Entities/Objects/Misc/paper.yml:33]: Test paper proto

File: Resources/Prototypes/_CP14/tags.yml [hunk 2]
```diff
@@ -9,3 +9,6 @@
 
 - type: Tag
   id: CP14CartridgeBulletRifle
+
+- type: Tag
+  id: CP14Bonfire
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/tags.yml:14]:
```suggestion
  id: CP14FitInBonfire
```

**Review by TheShuEd — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Misc/paper.yml (hunk 1 above)
Comment (Litogin) [reply Resources/Prototypes/Entities/Objects/Misc/paper.yml:33]: Ok

**Pull request merged**